constants/backend.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path')
+const { snakeCase, deepFreeze } = require('@keg-hub/jsutils')
+const { execSync } = require('child_process')
+
+const rootDir = path.join(__dirname, '../')
+const reposPath = path.join(rootDir, 'repos')
+
+// list of the herkin repo names located at `<root>/repos`
+const repos = execSync('ls', { cwd: reposPath })
+  .toString()
+  .split('\n')
+  .filter(Boolean)
+
+// object of env names to repo paths, e.g. { MESSENGER_PATH: <path>, ...}
+const repoPaths = repos.reduce(
```

Comment:
Ahh yah.... I love all of this.
Fully dynamic, and all I have to do is create a new folder, then BAM; brand new ENV. 
**Awesome**